src/visualization/types/SingleStat/options.tsx

```diff
@@ -46,6 +52,42 @@ const SingleStatOptions: FC<Props> = ({properties, update}) => {
     update({colors})
   }
 
+  const updateThreshold = useCallback(
+    (threshold: string) => {
+      if (threshold === THRESHOLD_TYPE_BG) {
+        update({
+          colors: properties.colors.map(color => {
+            if (color.type !== 'scale') {
+              return {
+                ...color,
+                type: THRESHOLD_TYPE_BG,
```

Comment:
...be converted to a ternary here?

I think that would eliminate a bunch of duplicate code

Reply:
totes can!